src/ucp/tag/rndv.h

```diff
@@ -33,6 +33,8 @@ typedef struct {
     uintptr_t                 sreq_ptr; /* request on the rndv initiator side - sender */
     uintptr_t                 rreq_ptr; /* request on the rndv receiver side */
     uint64_t                  address;  /* holds the address of the data buffer on the receiver's side */
+    size_t                    size;     /* size of the data to receive */
+    size_t                    offset;   /* offset of the data in the recv buffer */
```

Comment:
instead of adding offset, can we update address to reflect the offset?

Reply:
@bureddy can you pls remind why we need "offset"? if it is in order to identify offset in send buffer, maybe sreq_ptr of each fragment could point to send fragments requests which should already contain the offset?

Reply:
we do not have that info at the receiver side when sending RTR. send fragment requests at the sender side allocated only after RTR is received at the sender side if it is pipeline fragmentation requested from the receiver.